SwaggerGenerator

```diff
@@ -1,5 +1,5 @@
-// 20171013134624
-// http://localhost:49931/
+// 20171013142214
```

Comment:
I tweaked NSwag.Sample.NetCoreAngular to produce swagger docs using WebApi2Swagger and AspNetCore2Swagger . The app doesn't have use attributes outside the ones that come with Mvc in (i.e. NSwag.Annotations and Swashbuckle annotations), but the results look promising for a first go. 

Changes I made: https://github.com/pranavkm/NSwag/commit/d14264a7d68090b3c0d1c2babd357b38790f5a5e#diff-307d7217c51ea738573b49780ffd66c9.